tools/decoder.py

```diff
@@ -125,6 +126,7 @@ def print_all_addresses(addresses):
     def format_address(address):
         return "\n".join(format_addresses(elf, [address]))
 
+    lastepc = 0
```

Comment:
Why no repeats?

Reply:
It aesthetically avoid this by removing the second one from the display.
```
Fatal exception 29(StoreProhibitedCause):
epc1=0x40206b4b: _printf_i at /workdir/repo/newlib/newlib/libc/stdio/nano-vfprintf_i.c:189 (discriminator 1)
Exception (29) - StoreProhibited: A store referenced a page mapped with an attribute that does not permit stores
epc1=0x40206b4b: _printf_i at /workdir/repo/newlib/newlib/libc/stdio/nano-vfprintf_i.c:189 (discriminator 1)     (<- repetition)
sp: 3ffffda0 end: 3fffffd0 offset: 0150
```